SPECS/rsyslog/rsyslog.spec

```diff
@@ -128,6 +129,10 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %check
 %make_build check
 
+%pre
+groupadd --system syslog
+useradd --system --comment 'System Logging'  --gid syslog --shell /bin/false syslog
```

Comment:
These commands will fail if the group/user are already created. To be on the safe side, can we gate these with `getent` checks, similarly to how you do the `%postun` section?

Reply:
change as suggested